docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1178,15 +1178,83 @@ paths:
+          in: header
           description: |
-            The amount of uncommitted events Nakadi will stream before pausing the stream. When in paused
-            state and commit comes - the stream will resume. Minimal value is 1.
-          type: integer
-          format: int32
-          required: false
-          default: 10
+            The flow id of the request, which is written into the logs and passed to called services. Helpful
+            for operational troubleshooting and log analysis.
+          type: string
+      responses:
+        '200':
+          description: |
+            Ok. Stream started.
+            Stream format is a continuous series of `SubscriptionEventStreamBatch`s separated by `\n`
+          schema:
+            $ref: '#/definitions/SubscriptionEventStreamBatch'
+          headers:
+            X-Nakadi-StreamId:
+              description: |
+                the id of this stream generated by Nakadi. Must be used for committing events that were read by client
+                from this stream.
+              type: string
+        '400':
+          description: Bad Request
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Subscription not found.
+          schema:
+            $ref: '#/definitions/Problem'
+        '409':
+          description: |
+            Conflict. There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            already equals the maximal value - the total amount of this subscription partitions.
+
+            This status code is also returned in the case of resetting subscription cursors request still in the
+            progress.
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Access is forbidden for the client or event type
+          schema:
+            $ref: '#/definitions/Problem'
+
+    post:
+      tags:
+        - subscription-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        GET with body.
+        Starts a new stream for reading events from this subscription. Using this endpoint it is possible to specify
+        which partitions of subscription to read in this stream. If at least one of these partitions is already
+        assigned to another stream - the request will fail. It is possible to specify only the partitions that belong
+        to this subscription.
+        The rest of behavior is identical to endpoint GET /subscriptions/{subscription_id}/events.
+      parameters:
+        - name: partitions
+          in: body
+          schema:
+            type: object
+            properties:
+              items:
+                description: |
+                  List of partitions to read from in this stream.
+                  If not specified or if `items` array is empty - Nakadi will automatically choose partitions to assign
+                  to this stream.
```

Comment:
How? And how many?

Reply:
I provided some ideas in the comment above. Probably we need to discuss it with team to come to an agreement regarding this point.